modules/registry-accessing-directly.adoc

```diff
@@ -96,30 +73,27 @@ In the following examples, use:
 .. Pull an arbitrary image:
 +
 ----
-$ podman-pull name.io/image
+$ podman pull name.io/image
 ----
 
 .. Tag the new image with the form `<registry_ip>:<port>/<project>/<image>`.
 The project name must appear in this pull specification for {product-title} to
 correctly place and later access the image in the registry:
 +
 ----
-$ podman-tag name.io/image 172.30.124.220:5000/openshift/image
+$ podman tag name.io/image image-registry.openshift-image-registry.svc:5000/openshift/image
 ----
 +
 [NOTE]
 ====
-Your regular user must have the *system:image-builder* role for the specified
+You must have the `system:image-builder` role for the specified
 project, which allows the user to write or push an image. Otherwise, the
-`podman-push` in the next step will fail. To test, you can create a new project
+`podman push` in the next step will fail. To test, you can create a new project
 to push the image.
 ====
 
 .. Push the newly-tagged image to your registry:
 +
 ----
-$ podman-push 172.30.124.220:5000/openshift/image
-...
-cf2616975b4a: Image successfully pushed
-Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
+$ podman push name.io/image image-registry.openshift-image-registry.svc:5000/openshift/image
```

Comment:
This command should be:
$ podman push image-registry.openshift-image-registry.svc:5000/openshift/image

Reply:
Sorry for missng it before.

Reply:
Thanks so much, @wzheng1!